examples/player/simid_player.html

```diff
@@ -145,6 +154,13 @@
           
           adParams = JSON.stringify(adParamsObj);
           break;
+        case 'testers_nonlinear.html':
+          document.getElementById("nonlinear_ad").click();
```

Comment:
if you change the listeners from onclick to onchange, then use this line to set the new value, instead of calling "click", does it still work? Imo that would be a bit better than triggering a fake click

Reply:
onchange doesn't get triggered if we set the value to be checked with javascript so we'd have to simulate a click

Reply:
Agreed, clicking seems a little weird